packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -547,6 +549,12 @@ class SemanticsData with Diagnosticable {
   /// The reading direction is given by [textDirection].
   final String tooltip;
 
+  /// Indicates that this subtree represents a heading.
+  ///
+  /// The value should be a number between 1 and 6, indicating the hierarchical
```

Comment:
maybe mention 0 means this is not a heading

Reply:
Ok, done.